src/ansible/roles/keycloak/tasks/main.yml

```diff
@@ -1,15 +1,19 @@
+#- name: Get Keycloak latest version info
+#  uri:
+#    url: https://api.github.com/repos/keycloak/keycloak/releases/latest
+#    method: GET
+#    return_content: yes
+#    status_code: 200
+#    body_format: json
+#  register: result
```

Comment:
I think you can just remove this comment as the change will be visible in git history.

Reply:
done.